dask/dataframe/tests/test_groupby.py

```diff
@@ -107,11 +107,11 @@ def auto_shuffle_method(shuffle_method):
 
 
 @contextlib.contextmanager
-def groupby_axis_deprecated(*contexts):
+def groupby_axis_deprecated(*contexts, dask_op=True):
```

Comment:
I'm not sure I'm following what `dask_op` is for: Do we only raise the deprecation warning if it's not triggered by some Dask internal?

Reply:
Dask won't raise if dask-expr is enabled, since we only do the deprecation in dask/dask, while pandas raises the warning regardless.